checker/src/main/java/org/checkerframework/checker/initialization/InitializationVisitor.java

```diff
@@ -273,8 +273,10 @@ public void processClassTree(ClassTree node) {
         Store store = atypeFactory.getRegularExitStore(block);
 
         // Add field values for fields with an initializer.
-        for (Pair<VariableElement, Value> t : store.getAnalysis().getFieldValues()) {
-          store.addInitializedField(t.first);
+        for (FieldValues<Value> fieldValues : store.getAnalysis().getFieldValues()) {
+          if (fieldValues.initializer != null) {
+            store.addInitializedField(fieldValues.field.getField());
```

Comment:
This construct `field.getField()` is a bit wierd.  If it's already a field, why is `getField()` needed?
Hovering over the `field` field access indicates its value, but renaming the field from `field` to `fieldAccess` would eliminate the need to do so.
However, if you strongly prefer the name `field`, then the code can stand as is.